nixos/modules/services/networking/networkmanager.nix

```diff
@@ -283,6 +284,81 @@ in {
         '';
       };
 
+      rc-manager = mkOption {
+        type = types.enum [ "symlink" "file" "resolvconf" "netconfig" "unmanaged" "none" ];
+        default = "resolvconf";
+        description = ''
+          Set the <literal>resolv.conf</literal> management mode.
+          </para>
+          <para>
+          Options:
+          <variablelist>
+          <varlistentry>
```

Comment:
This is pretty much documented at https://developer.gnome.org/NetworkManager/stable/NetworkManager.conf.html
"`main` section" "rc-manager". Can't we link there?

Reply:
Sure! For consistency, we should do this for dns as well.